### DISCUSSION TITLE
crashing android app after importing some .js files

### DISCUSSION THREAD

**Discussion by HackerHack224** [react-native-elements/react-native-elements#3883] [Q&A] [answered]
Hello everyone, I'm new to mobile application development. I recently became acquainted with the React-native cross-platform framework. I'm familiar with JavaScript, which is why I chose this framework.
 
I wanted to ask about the problem that when I wanted to do it so as not to load App.js with many lines, I wanted to divide the elements into each part, like a separate banner, a separate header, a separate menu, etc. The problem is that when I want to add the "Main.js" element see code in bellow

App.js

`import { Platform, ScrollView, StatusBar, TouchableOpacity } from 'react-native';
// import SplashScreen from './splashScreen';
**import Main from './main'**;
// import axios from 'axios';
import React, { useEffect, useState } from 'react';
import { Svg, Path } from 'react-native-svg';
import { StyleSheet, Text, View, Button, SafeAreaView, TextInput, Image } from 'react-native';
import styles from './styles';

export default function App() {
  return (
    <View style={{ flex: 1 }}>
      <Main />
    </View>
  );
}`

This problem
`return (
    <View style={{ flex: 1 }}>
      <Main /> **{/*after adding this tag my app crashing*/}**
    </View>
  );`

//main.js

`import { Platform, ScrollView, StatusBar, TouchableOpacity } from 'react-native';
import SplashScreen from './splashScreen';
import axios from 'axios';
import React, { useEffect, useState } from 'react';
import { Svg, Path } from 'react-native-svg';
import { StyleSheet, Text, View, Button, SafeAreaView, TextInput, Image } from 'react-native';
import styles from './styles';

const Main = () => {
    const [response, setViewColor] = useState('#f0f0f0');
    const SBC = () => {

        console.log(StatusBar.setHidden)

        axios.post('http://zq.zzz.com.ua/version.php')
            .then(response => {
                console.log(response.data)
                if (response.data) {
                    setViewColor('green')
                }
            })
            .catch(error => {
                console.error(error)
            })
    }

    // css active
    const [isActive, setIsActive] = useState(false);
    const [textColor, setTextColor] = useState('white');

    const handlePress = () => {
        // Обработка нажатия
        setIsActive(true);
        setTextColor('red')

        // Дополнительная логика при необходимости
    };

    const handleRelease = () => {
        // Обработка отпускания
        setIsActive(false);
        // Дополнительная логика при необходимости
    };
    return (
        <SafeAreaView style={styles.container}>
            {Platform.select({
                ios: (
                    <View style={{
                        position: "absolute",
                        top: 0,
                        left: 0,
                        width: "100%",
                        height: 48,
                        backgroundColor: response,
                    }}>

                    </View>
                ),
                android: (
                    useEffect(() => {
                        // Установите цвет статусбара при монтировании компонента
                        StatusBar.setBackgroundColor('#f0f0f0'); // Задайте нужный цвет

                    }, [])
                )
            })}
            <ScrollView>
                {/* heaer start */}
                <View style={styles.header}>
                    <TextInput style={styles.inputHeader} placeholder="Поиск по Kaspi.kz" readOnly></TextInput>
                </View>
                {/* header end */}

                {/* recomedation banner start */}
                <View style={styles.banner}>
                    <ScrollView style={styles.bannerScroll} horizontal={true} showsHorizontalScrollIndicator={false}>

                        {/* banner 1 */}
                        <TouchableOpacity style={styles.mainBanners1}>
                            {/* <Image source={require('./assets/app-assets/banner1.png')} style={{ width: 166, height: 104, borderRadius: 8, marginBottom: 10 }} /> */}
                            <Text style={styles.txtBanner}>
                                29 января - 4 февраля
                            </Text>
                        </TouchableOpacity>

                        {/* banner 2 */}
                        <TouchableOpacity style={styles.mainBanners}>
                            {/* <Image source={require('./assets/app-assets/banner2.png')} style={{ width: 166, height: 104, borderRadius: 8, marginBottom: 10 }} /> */}
                            <Text style={styles.txtBanner}>
                                8 января - 4 февраля
                            </Text>
                        </TouchableOpacity>

                        {/* banner 3 */}
                        <TouchableOpacity style={styles.mainBanners}>
                            {/* <Image source={require('./assets/app-assets/banner3.png')} style={{ width: 166, height: 104, borderRadius: 8, marginBottom: 10 }} /> */}
                            <Text style={styles.txtBanner}>
                                29 января - 4 февраля
                            </Text>
                        </TouchableOpacity>

                        {/* banner 4 */}
                        <TouchableOpacity style={styles.mainBanners}>
                            {/* <Image source={require('./assets/app-assets/banner4.png')} style={{ width: 166, height: 104, borderRadius: 8, marginBottom: 10 }} /> */}
                            <Text style={styles.txtBanner}>
                                29 января - 4 февраля
                            </Text>
                        </TouchableOpacity>

                        {/* banner 5 */}
                        <TouchableOpacity style={styles.mainBanners}>
                            {/* <Image source={require('./assets/app-assets/banner5.png')} style={{ width: 166, height: 104, borderRadius: 8, marginBottom: 10 }} /> */}
                            <Text style={styles.txtBanner}>
                                8 января - 4 февраля
                            </Text>
                        </TouchableOpacity>

                        {/* banner 6 */}
                        <TouchableOpacity style={styles.mainBanners}>
                            {/* <Image source={require('./assets/app-assets/banner6.png')} style={{ width: 166, height: 104, borderRadius: 8, marginBottom: 10 }} /> */}
                            <Text style={styles.txtBanner}>
                                8 января - 4 февраля
                            </Text>
                        </TouchableOpacity>

                        {/* banner 7 */}
                        <TouchableOpacity style={styles.mainBanners}>
                            {/* <Image source={require('./assets/app-assets/banner7.png')} style={{ width: 166, height: 104, borderRadius: 8, marginBottom: 10 }} /> */}
                            <Text style={styles.txtBanner}>
                                29 января - 4 февраля
                            </Text>
                        </TouchableOpacity>

                        {/* banner 8 */}
                        <TouchableOpacity style={styles.mainBanners}>
                            {/* <Image source={require('./assets/app-assets/banner8.png')} style={{ width: 166, height: 104, borderRadius: 8, marginBottom: 10 }} /> */}
                            <Text style={styles.txtBanner}>
                                8 января - 4 февраля
                            </Text>
                        </TouchableOpacity>

                        {/* banner 9 */}
                        <TouchableOpacity style={styles.mainBanners}>
                            {/* <Image source={require('./assets/app-assets/banner9.png')} style={{ width: 166, height: 104, borderRadius: 8, marginBottom: 10 }} /> */}
                            <Text style={styles.txtBanner}>
                                29 января - 4 февраля
                            </Text>
                        </TouchableOpacity>

                        {/* banner 10 */}
                        <TouchableOpacity style={styles.mainBanners}>
                            {/* <Image source={require('./assets/app-assets/banner10.png')} style={{ width: 166, height: 104, borderRadius: 8, marginBottom: 10 }} /> */}
                            <Text style={styles.txtBanner}>
                                29 января - 4 февраля
                            </Text>
                        </TouchableOpacity>

                        {/* banner 11 */}
                        <TouchableOpacity style={styles.mainBanners}>
                            {/* <Image source={require('./assets/app-assets/banner11.png')} style={{ width: 166, height: 104, borderRadius: 8, marginBottom: 10 }} /> */}
                            <Text style={styles.txtBanner}>
                                29 января - 4 февраля
                            </Text>
                        </TouchableOpacity>

                        {/* banner 12 */}
                        <TouchableOpacity style={styles.mainBanners}>
                            {/* <Image source={require('./assets/app-assets/banner12.png')} style={{ width: 166, height: 104, borderRadius: 8, marginBottom: 10 }} /> */}
                            <Text style={styles.txtBanner}>
                                29 января - 4 февраля
                            </Text>
                        </TouchableOpacity>

                    </ScrollView>
                </View>
                {/* recomendation banner end */}

                {/* services start */}
                <View style={styles.services}>

                    {/* container 1 */}
                    <View style={styles.container1}>

                        <TouchableOpacity style={styles.sBox}>
                            {/* <Image source={require('./assets/app-assets/QR.png')} style={styles.sImg} /> */}
                            <Text style={[styles.sTxt]}>
                                Kaspi QR
                            </Text>
                        </TouchableOpacity>

                        <TouchableOpacity onPressIn={handlePress} onPressOut={handleRelease} style={styles.sBox}>
                            {/* <Image source={require('./assets/app-assets/Bank.png')} style={[styles.sImg]} /> */}
                            <Text style={styles.sTxt}>
                                Мой Банк
                            </Text>
                        </TouchableOpacity>

                        <TouchableOpacity style={styles.sBox}>
                            {/* <Image source={require('./assets/app-assets/Payments.png')} style={styles.sImg} /> */}
                            <Text style={styles.sTxt}>
                                Платежи
                            </Text>
                        </TouchableOpacity>

                        <TouchableOpacity style={styles.sBox}>
                            {/* <Image source={require('./assets/app-assets/Transfers.png')} style={styles.sImg} /> */}
                            <Text style={styles.sTxt}>
                                Переводы
                            </Text>
                        </TouchableOpacity>

                    </View>

                    {/* container 2 */}
                    <View style={styles.container2}>

                        <TouchableOpacity style={styles.sBox}>
                            {/* <Image source={require('./assets/app-assets/Shop.png')} style={styles.sImg} /> */}
                            <Text style={styles.sTxt}>
                                Магазин
                            </Text>
                        </TouchableOpacity>

                        <TouchableOpacity style={styles.sBox}>
                            {/* <Image source={require('./assets/app-assets/Travel.png')} style={styles.sImg} /> */}
                            <Text style={styles.sTxt}>
                                Travel
                            </Text>
                        </TouchableOpacity>

                        <TouchableOpacity style={styles.sBox}>
                            {/* <Image source={require('./assets/app-assets/Gov.png')} style={styles.sImg} /> */}
                            <Text style={styles.sTxt}>
                                Госуслуги
                            </Text>
                        </TouchableOpacity>

                        <TouchableOpacity style={styles.sBox}>
                            {/* <Image source={require('./assets/app-assets/Ad.png')} style={styles.sImg} /> */}
                            <Text style={styles.sTxt}>
                                Объявления
                            </Text>
                        </TouchableOpacity>

                    </View>
                </View>
                {/* services end */}

                {/* market start */}
                <TouchableOpacity style={styles.market}>
                    {/* <Image source={require('./assets/app-assets/magnum.png')} style={{ height: 42, width: 42, marginLeft: 15, marginRight: 10 }} /> */}

                    <View style={styles.marketTxt}>
                        <Text style={styles.marketTxt1}>
                            Magnum
                        </Text>

                        <Text style={styles.marketTxt2}>
                            Продукты питания с бесплатной доставкой
                        </Text>
                    </View>

                    {/* <Image source={require('./assets/app-assets/arrow.png')} style={{ height: 14, width: 8, position: 'absolute', right: 15 }} /> */}

                </TouchableOpacity>
                {/* market end */}

                {/* online credit start */}
                <View style={styles.oCredit}>
                    <ScrollView style={styles.creditS} horizontal={true} showsHorizontalScrollIndicator={false}>

                        <View style={styles.oCreditS}>

                            <TouchableOpacity style={styles.oCreditSelect1}>
                                {/* <Image source={require('./assets/app-assets/Kredit.png')} style={{ height: 41, width: 54, marginRight: 10, marginLeft: 20 }} /> */}
                                <View style={styles.txtCredit}>
                                    <Text style={styles.txtCredit1}>
                                        Кредит на Покупки
                                    </Text>
                                    <Text style={styles.txtCredit2}>
                                        Кредит или рассрочка 0%
                                    </Text>
                                </View>
                            </TouchableOpacity>

                            <TouchableOpacity style={styles.oCreditSelect2}>
                                {/* <Image source={require('./assets/app-assets/DepositNew.png')} style={{ height: 41, width: 54, marginRight: 10, marginLeft: 20 }} /> */}
                                <View style={styles.txtCredit}>
                                    <Text style={styles.txtCredit1}>
                                        Kaspi Депозит
                                    </Text>
                                    <Text style={styles.txtCredit2}>
                                        Эффективная ставка 15%
                                    </Text>
                                </View>
                            </TouchableOpacity>
                        </View>

                        <View style={styles.oCreditS1}>
                            <TouchableOpacity style={styles.oCreditSelect1}>
                                {/* <Image source={require('./assets/app-assets/KN.png')} style={{ height: 41, width: 54, marginRight: 10, marginLeft: 5 }} /> */}
                                <View style={styles.txtCredit}>
                                    <Text style={styles.txtCredit1}>
                                        Кредит Наличными
                                    </Text>
                                    <Text style={styles.txtCredit2}>
                                        До 1 млн тенге
                                    </Text>
                                </View>
                            </TouchableOpacity>

                            <TouchableOpacity style={styles.oCreditSelect2}>
                                {/* <Image source={require('./assets/app-assets/Red.png')} style={{ height: 41, width: 54, marginRight: 10, marginLeft: 5 }} /> */}
                                <View style={styles.txtCredit}>
                                    <Text style={styles.txtCredit1}>
                                        Kaspi Red
                                    </Text>
                                    <Text style={styles.txtCredit2}>
                                        Рассрочка 0%
                                    </Text>
                                </View>
                            </TouchableOpacity>
                        </View>

                    </ScrollView>
                </View>
                {/* online credit end */}

                {/* empty start */}
                <View style={styles.empty}>

                </View>
                {/* empty end */}

                <Button color="#00ff00" title="button" onPress={SBC} />
            </ScrollView>
            {/* footer start */}
            {Platform.select({
                ios: (
                    <View style={styles.footerI}>
                        <View style={styles.IOSFooter}>

                            <TouchableOpacity style={styles.footerSelect}>
                                <Svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 28 28" fill="none">
                                    <Path
                                        fill-rule="evenodd"
                                        clip-rule="evenodd"
                                        d="M12.95 2.82913L4.20002 9.39163C3.75935 9.72212 3.50002 10.2408 3.50002 10.7916V23.9166C3.50002 24.8831 4.28352 25.6666 5.25002 25.6666H9.91668L10.0852 25.6586C10.9726 25.5738 11.6667 24.8263 11.6667 23.9166V18.6666H16.3333V23.9166C16.3333 24.8831 17.1168 25.6666 18.0833 25.6666H22.75C23.7165 25.6666 24.5 24.8831 24.5 23.9166V10.7916C24.5 10.2408 24.2407 9.72212 23.8 9.39163L15.05 2.82913C14.4278 2.36246 13.5722 2.36246 12.95 2.82913ZM13.9988 4.95713L22.1667 11.0833V23.3333H18.6667V18.0833C18.6667 17.1168 17.8832 16.3333 16.9167 16.3333H11.0833L10.9148 16.3413C10.0274 16.4261 9.33335 17.1736 9.33335 18.0833V23.3333H5.83335V11.0821L13.9988 4.95713ZM10.5 12.8333H17.5C18.1443 12.8333 18.6667 13.3556 18.6667 14C18.6667 14.6443 18.1443 15.1666 17.5 15.1666H10.5C9.85568 15.1666 9.33335 14.6443 9.33335 14C9.33335 13.3556 9.85568 12.8333 10.5 12.8333Z"
                                        fill="#F14635"
                                    />
                                </Svg>
                                <Text style={styles.txtFooter}>
                                    Главная
                                </Text>
                            </TouchableOpacity>

                            <TouchableOpacity style={styles.footerSelect}>
                                <Svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 28 28" fill="none">
                                    <Path
                                        fillRule="evenodd"
                                        clipRule="evenodd"
                                        d="M1.16667 2.91663C1.16667 1.95013 1.95017 1.16663 2.91667 1.16663H8.16667C8.811 1.16663 9.33334 1.68896 9.33334 2.33329C9.33334 2.97762 8.811 3.49996 8.16667 3.49996H3.50001V8.16663C3.50001 8.81096 2.97767 9.33329 2.33334 9.33329C1.68901 9.33329 1.16667 8.81096 1.16667 8.16663V2.91663ZM18.6667 2.33329C18.6667 1.68896 19.189 1.16663 19.8333 1.16663H25.0833C26.0498 1.16663 26.8333 1.95013 26.8333 2.91663V8.16663C26.8333 8.81096 26.311 9.33329 25.6667 9.33329C25.0223 9.33329 24.5 8.81096 24.5 8.16663V3.49996H19.8333C19.189 3.49996 18.6667 2.97762 18.6667 2.33329ZM2.33334 18.6666C2.97767 18.6666 3.50001 19.189 3.50001 19.8333V24.5H8.16667C8.811 24.5 9.33334 25.0223 9.33334 25.6666C9.33334 26.311 8.811 26.8333 8.16667 26.8333H2.91667C1.95018 26.8333 1.16667 26.0498 1.16667 25.0833V19.8333C1.16667 19.189 1.68901 18.6666 2.33334 18.6666ZM25.6667 18.6666C26.311 18.6666 26.8333 19.189 26.8333 19.8333V25.0833C26.8333 26.0498 26.0498 26.8333 25.0833 26.8333H19.8333C19.189 26.8333 18.6667 26.311 18.6667 25.6666C18.6667 25.0223 19.189 24.5 19.8333 24.5H24.5V19.8333C24.5 19.189 25.0223 18.6666 25.6667 18.6666ZM7.58336 7.58325V11.0833H11.0834V7.58325H7.58336ZM6.41669 5.24992C5.77236 5.24992 5.25003 5.77225 5.25003 6.41658V12.2499C5.25003 12.8943 5.77236 13.4166 6.41669 13.4166H12.25C12.8944 13.4166 13.4167 12.8943 13.4167 12.2499V6.41659C13.4167 5.77225 12.8944 5.24992 12.25 5.24992H6.41669ZM16.9167 11.0833V7.58325H20.4167V11.0833H16.9167ZM14.5834 6.41658C14.5834 5.77225 15.1057 5.24992 15.75 5.24992H21.5834C22.2277 5.24992 22.75 5.77225 22.75 6.41659V12.2499C22.75 12.8943 22.2277 13.4166 21.5834 13.4166H15.75C15.1057 13.4166 14.5834 12.8943 14.5834 12.2499V6.41658ZM7.58336 16.9166V20.4166H11.0834V16.9166H7.58336ZM6.41669 14.5833C5.77236 14.5833 5.25003 15.1056 5.25003 15.7499V21.5833C5.25003 22.2276 5.77236 22.7499 6.41669 22.7499H12.25C12.8944 22.7499 13.4167 22.2276 13.4167 21.5833V15.7499C13.4167 15.1056 12.8944 14.5833 12.25 14.5833H6.41669ZM15.75 14.5833H20.0279V16.9166H16.9167V20.028H14.5834V15.7499C14.5834 15.1056 15.1057 14.5833 15.75 14.5833ZM22.75 17.3058H20.4167V20.4166H17.3057V22.7499H21.5834C22.2277 22.7499 22.75 22.2276 22.75 21.5833V17.3058ZM8.45834 8.45829H10.2083V10.2083H8.45834V8.45829ZM19.5417 8.45833H17.7917V10.2083H19.5417V8.45833ZM17.7917 17.7917H19.5417V19.5417H17.7917V17.7917ZM10.2083 17.7917H8.45832V19.5417H10.2083V17.7917Z"
                                        fill="#A8A8A8"
                                    />
                                </Svg>
                                <Text style={styles.txtFooter1}>
                                    Kaspi QR
                                </Text>
                            </TouchableOpacity>

                            <TouchableOpacity style={styles.footerSelect}>
                                <Svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 28 28" fill="none">
                                    <Path
                                        fillRule="evenodd"
                                        clipRule="evenodd"
                                        d="M7.08643 2.33337C4.46008 2.33337 2.33334 4.46138 2.33334 7.08646V16.5926C2.33334 19.0721 4.23053 21.1079 6.65433 21.3263V23.5927C6.65433 24.8034 8.03502 25.4943 9.00387 24.7679L13.5679 21.3457H20.9136C23.5399 21.3457 25.6667 19.2177 25.6667 16.5926V7.08646C25.6667 4.46138 23.5399 2.33337 20.9136 2.33337H7.08643ZM4.92594 7.08646C4.92594 5.89334 5.89478 4.92597 7.08643 4.92597H20.9136C22.1052 4.92597 23.0741 5.89334 23.0741 7.08646V16.5926C23.0741 17.7858 22.1052 18.7531 20.9136 18.7531H13.1932C12.8759 18.7531 12.5653 18.8563 12.3121 19.047L9.24692 21.3457V20.2222C9.24692 19.411 8.58865 18.7531 7.77846 18.7531H7.08643C5.89478 18.7531 4.92594 17.7858 4.92594 16.5926V7.08646ZM14 14C14.9554 14 15.7284 13.2261 15.7284 12.2716C15.7284 11.3171 14.9554 10.5433 14 10.5433C13.0447 10.5433 12.2716 11.3171 12.2716 12.2716C12.2716 13.2261 13.0447 14 14 14ZM20.4612 13.4382C20.1439 13.7836 19.6916 14 19.1852 14C18.2299 14 17.4568 13.2261 17.4568 12.2716C17.4568 11.3171 18.2299 10.5433 19.1852 10.5433C19.6882 10.5433 20.1439 10.7589 20.4579 11.103C20.7414 11.4106 20.9136 11.821 20.9136 12.2716C20.9136 12.7213 20.7414 13.1308 20.4612 13.4382ZM8.81483 14C9.77017 14 10.5432 13.2261 10.5432 12.2716C10.5432 11.3171 9.77017 10.5433 8.81483 10.5433C7.85948 10.5433 7.08643 11.3171 7.08643 12.2716C7.08643 13.2261 7.85948 14 8.81483 14Z"
                                        fill="#A8A8A8"
                                    />
                                </Svg>
                                <Text style={styles.txtFooter1}>
                                    Сообщения
                                </Text>
                            </TouchableOpacity>

                            <TouchableOpacity style={styles.footerSelect}>
                                <Svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 28 28" fill="none">
                                    <Path
                                        fillRule="evenodd"
                                        clipRule="evenodd"
                                        d="M2.4017 4.66663H25.665V6.99996H2.4017V4.66663ZM2.40169 12.8332H25.665V15.1665H2.40169V12.8332ZM25.665 20.9999H2.40169V23.3332H25.665V20.9999Z"
                                        fill="#A8A8A8"
                                    />
                                </Svg>
                                <Text style={styles.txtFooter1}>
                                    Меню
                                </Text>
                            </TouchableOpacity>

                        </View>
                    </View>
                ),

                android: (
                    <View style={styles.footerA}>

                        <View style={styles.footerSelect}>
                            <Svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 28 28" fill="none">
                                <Path
                                    fill-rule="evenodd"
                                    clip-rule="evenodd"
                                    d="M12.95 2.82913L4.20002 9.39163C3.75935 9.72212 3.50002 10.2408 3.50002 10.7916V23.9166C3.50002 24.8831 4.28352 25.6666 5.25002 25.6666H9.91668L10.0852 25.6586C10.9726 25.5738 11.6667 24.8263 11.6667 23.9166V18.6666H16.3333V23.9166C16.3333 24.8831 17.1168 25.6666 18.0833 25.6666H22.75C23.7165 25.6666 24.5 24.8831 24.5 23.9166V10.7916C24.5 10.2408 24.2407 9.72212 23.8 9.39163L15.05 2.82913C14.4278 2.36246 13.5722 2.36246 12.95 2.82913ZM13.9988 4.95713L22.1667 11.0833V23.3333H18.6667V18.0833C18.6667 17.1168 17.8832 16.3333 16.9167 16.3333H11.0833L10.9148 16.3413C10.0274 16.4261 9.33335 17.1736 9.33335 18.0833V23.3333H5.83335V11.0821L13.9988 4.95713ZM10.5 12.8333H17.5C18.1443 12.8333 18.6667 13.3556 18.6667 14C18.6667 14.6443 18.1443 15.1666 17.5 15.1666H10.5C9.85568 15.1666 9.33335 14.6443 9.33335 14C9.33335 13.3556 9.85568 12.8333 10.5 12.8333Z"
                                    fill="#F14635"
                                />
                            </Svg>
                            <Text style={styles.txtFooter}>
                                Главная
                            </Text>
                        </View>

                        <View style={styles.footerSelect}>
                            <Svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 28 28" fill="none">
                                <Path
                                    fillRule="evenodd"
                                    clipRule="evenodd"
                                    d="M1.16667 2.91663C1.16667 1.95013 1.95017 1.16663 2.91667 1.16663H8.16667C8.811 1.16663 9.33334 1.68896 9.33334 2.33329C9.33334 2.97762 8.811 3.49996 8.16667 3.49996H3.50001V8.16663C3.50001 8.81096 2.97767 9.33329 2.33334 9.33329C1.68901 9.33329 1.16667 8.81096 1.16667 8.16663V2.91663ZM18.6667 2.33329C18.6667 1.68896 19.189 1.16663 19.8333 1.16663H25.0833C26.0498 1.16663 26.8333 1.95013 26.8333 2.91663V8.16663C26.8333 8.81096 26.311 9.33329 25.6667 9.33329C25.0223 9.33329 24.5 8.81096 24.5 8.16663V3.49996H19.8333C19.189 3.49996 18.6667 2.97762 18.6667 2.33329ZM2.33334 18.6666C2.97767 18.6666 3.50001 19.189 3.50001 19.8333V24.5H8.16667C8.811 24.5 9.33334 25.0223 9.33334 25.6666C9.33334 26.311 8.811 26.8333 8.16667 26.8333H2.91667C1.95018 26.8333 1.16667 26.0498 1.16667 25.0833V19.8333C1.16667 19.189 1.68901 18.6666 2.33334 18.6666ZM25.6667 18.6666C26.311 18.6666 26.8333 19.189 26.8333 19.8333V25.0833C26.8333 26.0498 26.0498 26.8333 25.0833 26.8333H19.8333C19.189 26.8333 18.6667 26.311 18.6667 25.6666C18.6667 25.0223 19.189 24.5 19.8333 24.5H24.5V19.8333C24.5 19.189 25.0223 18.6666 25.6667 18.6666ZM7.58336 7.58325V11.0833H11.0834V7.58325H7.58336ZM6.41669 5.24992C5.77236 5.24992 5.25003 5.77225 5.25003 6.41658V12.2499C5.25003 12.8943 5.77236 13.4166 6.41669 13.4166H12.25C12.8944 13.4166 13.4167 12.8943 13.4167 12.2499V6.41659C13.4167 5.77225 12.8944 5.24992 12.25 5.24992H6.41669ZM16.9167 11.0833V7.58325H20.4167V11.0833H16.9167ZM14.5834 6.41658C14.5834 5.77225 15.1057 5.24992 15.75 5.24992H21.5834C22.2277 5.24992 22.75 5.77225 22.75 6.41659V12.2499C22.75 12.8943 22.2277 13.4166 21.5834 13.4166H15.75C15.1057 13.4166 14.5834 12.8943 14.5834 12.2499V6.41658ZM7.58336 16.9166V20.4166H11.0834V16.9166H7.58336ZM6.41669 14.5833C5.77236 14.5833 5.25003 15.1056 5.25003 15.7499V21.5833C5.25003 22.2276 5.77236 22.7499 6.41669 22.7499H12.25C12.8944 22.7499 13.4167 22.2276 13.4167 21.5833V15.7499C13.4167 15.1056 12.8944 14.5833 12.25 14.5833H6.41669ZM15.75 14.5833H20.0279V16.9166H16.9167V20.028H14.5834V15.7499C14.5834 15.1056 15.1057 14.5833 15.75 14.5833ZM22.75 17.3058H20.4167V20.4166H17.3057V22.7499H21.5834C22.2277 22.7499 22.75 22.2276 22.75 21.5833V17.3058ZM8.45834 8.45829H10.2083V10.2083H8.45834V8.45829ZM19.5417 8.45833H17.7917V10.2083H19.5417V8.45833ZM17.7917 17.7917H19.5417V19.5417H17.7917V17.7917ZM10.2083 17.7917H8.45832V19.5417H10.2083V17.7917Z"
                                    fill="#A8A8A8"
                                />
                            </Svg>
                            <Text style={styles.txtFooter1}>
                                Kaspi QR
                            </Text>
                        </View>

                        <View style={styles.footerSelect}>
                            <Svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 28 28" fill="none">
                                <Path
                                    fillRule="evenodd"
                                    clipRule="evenodd"
                                    d="M7.08643 2.33337C4.46008 2.33337 2.33334 4.46138 2.33334 7.08646V16.5926C2.33334 19.0721 4.23053 21.1079 6.65433 21.3263V23.5927C6.65433 24.8034 8.03502 25.4943 9.00387 24.7679L13.5679 21.3457H20.9136C23.5399 21.3457 25.6667 19.2177 25.6667 16.5926V7.08646C25.6667 4.46138 23.5399 2.33337 20.9136 2.33337H7.08643ZM4.92594 7.08646C4.92594 5.89334 5.89478 4.92597 7.08643 4.92597H20.9136C22.1052 4.92597 23.0741 5.89334 23.0741 7.08646V16.5926C23.0741 17.7858 22.1052 18.7531 20.9136 18.7531H13.1932C12.8759 18.7531 12.5653 18.8563 12.3121 19.047L9.24692 21.3457V20.2222C9.24692 19.411 8.58865 18.7531 7.77846 18.7531H7.08643C5.89478 18.7531 4.92594 17.7858 4.92594 16.5926V7.08646ZM14 14C14.9554 14 15.7284 13.2261 15.7284 12.2716C15.7284 11.3171 14.9554 10.5433 14 10.5433C13.0447 10.5433 12.2716 11.3171 12.2716 12.2716C12.2716 13.2261 13.0447 14 14 14ZM20.4612 13.4382C20.1439 13.7836 19.6916 14 19.1852 14C18.2299 14 17.4568 13.2261 17.4568 12.2716C17.4568 11.3171 18.2299 10.5433 19.1852 10.5433C19.6882 10.5433 20.1439 10.7589 20.4579 11.103C20.7414 11.4106 20.9136 11.821 20.9136 12.2716C20.9136 12.7213 20.7414 13.1308 20.4612 13.4382ZM8.81483 14C9.77017 14 10.5432 13.2261 10.5432 12.2716C10.5432 11.3171 9.77017 10.5433 8.81483 10.5433C7.85948 10.5433 7.08643 11.3171 7.08643 12.2716C7.08643 13.2261 7.85948 14 8.81483 14Z"
                                    fill="#A8A8A8"
                                />
                            </Svg>
                            <Text style={styles.txtFooter1}>
                                Сообщения
                            </Text>
                        </View>

                        <View style={styles.footerSelect}>
                            <Svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 28 28" fill="none">
                                <Path
                                    fillRule="evenodd"
                                    clipRule="evenodd"
                                    d="M2.4017 4.66663H25.665V6.99996H2.4017V4.66663ZM2.40169 12.8332H25.665V15.1665H2.40169V12.8332ZM25.665 20.9999H2.40169V23.3332H25.665V20.9999Z"
                                    fill="#A8A8A8"
                                />
                            </Svg>
                            <Text style={styles.txtFooter1}>
                                Меню
                            </Text>
                        </View>

                    </View>
                )
            })}
            {/* footer end */}

        </SafeAreaView>
    )
}

export default Main;`

please tell me what the problem is, maybe I didn’t install some plugin, maybe I need to add some or I made a mistake with the code

**Comment by HackerHack224** [comment]
By the way, I compiled my project to .apk i used a commands
./gradew clean
./gradlew assembleRelease

**Comment by theianmay** [comment, marked as answer]
Thank you for opening this discussion, and apologies for the delay. The new team and I have been working to restore the library to a working state. I believe your discussion is a more general mobile app development question, instead of something related to our UI library. For reference, React Native Elements is a UI toolkit that you can use when developing an app, specifically to bring in specific UI components like Buttons, Slider, etc. - If you have a question for React Native themselves, you'll have to navigate and post in Meta's repo. Hope that helps, thanks!
